template.distillery/Makefile.db

```diff
@@ -30,8 +30,20 @@ ELIOMDEP          := $(ENV_PSQL) $(ELIOMDEP)
 # should be the same as ENV_PSQL)
 OCSIGENSERVER     := $(OCSIGENSERVER)
 OCSIGENSERVER.OPT := $(OCSIGENSERVER.OPT)
+
 ## ---------------------------------------------------------------------
 
+##----------------------------------------------------------------------
+
+pg_dump = pg_dump
+
+pg_ctl  = $(shell \
+           sh -c "which pg_ctl" || \
+           sh -c "which pg_ctlcluster" || \
+           sh -c "if [ -f pg_ctl ]; then (echo ./pg_ctl) else (echo '') fi")
```

Comment:
Debian should be just like Ubuntu in that regard.

Wouldn't it be easier and more reliable to use the `pg_ctl` in `/usr/lib/postgresql/x.y/bin/pg_ctl`? I am not sure that going through `pg_ctlcluster` is of any use to us, given that we create a local directory and do not have anything to do with the Debian/Ubuntu conventions.